### PR TITLE
fix(create-analog): improve E2E testing for create-analog

### PR DESCRIPTION
**Pull request by Micha-Richter**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: Test cases

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] astro-angular
- [x] create-analog
- [ ] router
- [ ] platform
- [ ] content
- [ ] nx-plugin
- [ ] trpc

## What is the current behavior?

The test cases in create-analog-e2e use npm as package manager which doesn't play well with the default pnpm. Running the tests gives a lot of warnings and errors coming from not finding packages or version mismatches (use --skip-nx-cache while debugging the tests!).

Closes #

## What is the new behavior?

The tests use pnpm for installing the packages now. As a side effect tests are running much faster.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by Micha-Richter** [comment]
CI log:
```
  Node.js version v18.17.1 detected.
  The Angular CLI requires a minimum Node.js version of v18.19.
```

18.17.1 is defined in .node-version 7 months ago ("update to astro 3.0.x")!?
Can we increase this version?

**Comment by brandonroberts** [comment]
Yes, that can be a part of this PR. Let's update to 18.19.1

**Comment by Micha-Richter** [comment]
@brandonroberts i've rebased the pr on main to make it as smooth as possible for you - but somehow the beta branch was merged into the pr. don't know what went wrong 🙈
Sorry for the inconvenience 😐

File: apps/create-analog-e2e/tests/create-analog.spec.ts [hunk 1]
```diff
@@ -83,19 +86,23 @@ describe('create-analog e2e', () => {
       checkFilesExist(`${tmpDir}/dist/analog/public/index.html`)
     ).not.toThrow();
 
-    rmdirSync(tmpDir, { recursive: true });
+    fs.rmdirSync(tmpDir, { recursive: true });
   }, 120000);
 
   it.skip('should create my-app with the next release', async () => {
```
Comment (brandonroberts) [inline apps/create-analog-e2e/tests/create-analog.spec.ts:92]: Re-enable the test for the next version of Angular.

```suggestion
  it('should create my-app with the next release', async () => {
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks! @allcontributors add @Micha-Richter for code

**Pull request merged**